src/effects/VST3/VST3Effect.cpp

```diff
@@ -57,24 +59,170 @@ namespace {
    constexpr auto processorStateKey  = wxT("ProcessorState");
    constexpr auto controllerStateKey = wxT("ControllerState");
 
-Steinberg::Vst::IHostApplication& LocalContext()
+size_t VST3ProcessBlock(
```

Comment:
I cut this function and pasted it just before `VST3Effect::ProcessBlock` so that there are fewer difference lines when viewed with whitespace changes ignored.

This makes the commit more easily reviewable.  I suggest you do that before merging it.